testimony/__init__.py

```diff
@@ -209,6 +218,11 @@ def main(report, paths, json_output, nocolor):
     """
     SETTINGS['json'] = json_output
     SETTINGS['nocolor'] = nocolor
+    SETTINGS['input_tags'] = tags
```

Comment:
I purposefully called this `input_tags` to differentiate this between the `input_tags` and the tags present in test cases themselves.